builder/copy.go

```diff
@@ -84,8 +86,5 @@ func copyFile(src, dest string) error {
 }
 
 func debugPrint(message string) {
```

Comment:
I can re-use this function directly without re-factoring it to a common package. just need to expose it publicly, but I feels it wrong to use util/common function from a specific package.